<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>351</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="editProfileButton" >
     <property name="maximumSize" >
      <size>
       <width>77</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>&amp;Edit my profile</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>General Settings</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <widget class="QGroupBox" name="groupBox_4" >
        <property name="title" >
         <string>Idle status</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="2" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>minutes</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QSpinBox" name="notAvailableSpinBox" >
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="showNotAvailableCheckBox" >
           <property name="text" >
            <string>Show me as "not available" when I am inactive for:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="title" >
         <string>When I double-click on a contact </string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" >
          <widget class="QCheckBox" name="callCellPhoneCheckBox" >
           <property name="text" >
            <string>Call cell phone or land line if contact not available</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2" >
          <widget class="QRadioButton" name="startFreeCallRadioButton" >
           <property name="text" >
            <string>Start a free call if applicable</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2" >
          <widget class="QRadioButton" name="startChatOnlyRadioButton" >
           <property name="text" >
            <string>Start a chat if applicable</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>When I start my computer</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="startWengoPhoneCheckBox" >
           <property name="text" >
            <string>Automatically start WengoPhone</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>startWengoPhoneCheckBox</tabstop>
  <tabstop>startFreeCallRadioButton</tabstop>
  <tabstop>callCellPhoneCheckBox</tabstop>
  <tabstop>startChatOnlyRadioButton</tabstop>
  <tabstop>showNotAvailableCheckBox</tabstop>
  <tabstop>notAvailableSpinBox</tabstop>
  <tabstop>editProfileButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
